docs/source/guide/Basics.lrad

```diff
@@ -31,10 +32,55 @@ Strings are enclosed in double quotes:
 "this is a string"
 ```
 
+### Vectors
+
+For talking about sequences of values `radicle` has vectors. Literal notation
```

Comment:
Hm, the phrasing "For talking about sequences" is a little strange. How about something like "`radicle`'s *vectors* are an array-like container for values"?